scripts/site-types/yii.sh

```diff
@@ -71,7 +77,6 @@ block="server {
     location ~ \.php$ {
         fastcgi_split_path_info ^(.+\.php)(/.+)$;
         fastcgi_pass unix:/var/run/php/php$5-fpm.sock;
-        fastcgi_index index.php;
```

Comment:
I believe we still need this because if `index-test.php` is missing the `location ~ \.php$` won't have an `fastcgi_index` to match.

Reply:
@svpernova09 fixed